arch/ecmwf/aurora/nec/4.0.0/toolchain.cmake

```diff
@@ -0,0 +1,89 @@
+
+####################################################################
+# COMPILER
+####################################################################
+
+include( /opt/nec/ve/share/cmake/toolchainVE.cmake )
```

Comment:
Traditionally, the low-level toolchains (per compiler version) are symlink up to the `arch/toolchains/<arch-toolchain>-.cmake` files higher up, for re-use?